fpdf/table.py

```diff
@@ -501,6 +501,7 @@ def _render_table_cell(
                     wrapmode=self._wrapmode,
                     padding=padding,
                     link=cell.link,
+                    page_break_buffer=10,
```

Comment:
There should be no "magical numbers" like that in the code.
How did you arrive at exactly this size?
Are you aware that size specifications are relative to the document units? If someone uses inches as document units, this turns into half a page!